module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -449,6 +461,7 @@ namespace module::platform::darwin {
                                      MeasurementType::GYROSCOPE());
 
                 // Calculate accelerometer noise factor
+                // Tom has never seen this. Updating the noise dynamically based on the acceleration seems very spicy
                 Eigen::Matrix3d acc_noise =
                     config.motionFilter.noise.measurement.accelerometer
                     + ((sensors->accelerometer.norm() - std::abs(G)) * (sensors->accelerometer.norm() - std::abs(G)))
```

Comment:
Blame this one on me. It's a software engineers solution to a mechatronics problem :P 

Since our filter doesn't actually track the acceleration, it just uses the accelerometer as a down vector, however when it's going all over the place (and isn't 9.8m/s^2) what we do is if we don't have a vector that is 9.8m/s^2 we increase the "noise" so it has less impact on the state of the filter. At one point in the past doing this slightly improved our filter when the robot was walking and the noisy "hits" on the ground came in and messed with our filter.